File: 05_simple_interest.cpp
//C++ program to calculate simple interest

#include <iostream>
using namespace std;
int main() {
  
    float prin, rate, time, si; //declare variable

    // Get inputs from user
    cout << "Enter principal amount: ";
    cin >> prin;
    
    cout << "Enter time in years: ";
    cin >> time;
    
    cout << "Enter rate on interest: ";
    cin >> rate;

    // Calculate simple interest and display result
    si=(prin*rate*time)/100;
    
    cout << "Simple interest calculated is: " << si << end1;

    return 0;

}

File: 05_simple_interest.py
#python program to find simple interest

#take principal amount, rate of interest and time inputs from user
prin=float(input("Enter principal amount: "))
rate=float(input("Enter rate of interest (in percentage): "))
time=float(input("Enter time period (in years): "))

#calculate simple interest and display it
si=(prin*rate*time)/100
print("Simple interest calculated is: ", si)
